examples/test_cheque.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
```

Comment:
Also, can you add a docstring with a short/long description of what the example does?
See this for an example: https://github.com/JelteF/PyLaTeX/blob/master/examples/lists.py

(this doc string should be before the `begin-doc-include` comment)